packages/types/src/index.ts

```diff
@@ -209,6 +209,12 @@ export interface Datepicker extends Action {
   confirm?: Confirm;
 }
 
+export interface RadioButtons extends Action {
+  type: 'radio_buttons';
+  initial_option?: Option;
+  options: Option[];
+}
```

Comment:
Do we need to have `confirm` as well? or should we modify the api doc? https://api.slack.com/reference/block-kit/block-elements#radio

Reply:
🤦‍♂ You're right